src/utils/__tests__/isStandardSyntaxAtRule.test.js

```diff
@@ -2,58 +2,55 @@ import isStandardSyntaxAtRule from "../isStandardSyntaxAtRule"
 import less from "postcss-less"
 import postcss from "postcss"
 import scss from "postcss-scss"
-import test from "tape"
-
-test("isStandardSyntaxAtRule", t => {
-  t.plan(12)
 
+it("isStandardSyntaxAtRule", () => {
   atRules("@charset UTF-8;", atRule => {
-    t.ok(isStandardSyntaxAtRule(atRule), "non nested at-rules without quotes")
```

Comment:
In all cases where we *have* descriptions for assertions, we should preserve them. This means changing the `it()` above to `describe()`, and then wrapping these assertions in `it("description", () => {..})`.